pkg/scheduler/framework/parallelizer/errorflag.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package parallelizer
+
+// ErrorFlag is a flag for indicating whether an error has occurred when running tasks in
+// parallel with the parallelizer.
+type ErrorFlag struct {
```

Comment:
how does this work with Parallelizer?

Reply:
Hi Ryan! The error flag is designed to be used in doWork when calling parallelizer.ParallelizeUntil -> it allows exit on first error (in combination with context `cancel()`).